### PR TITLE
More links to API documentation

### PR DESCRIPTION
**Pull request by philipphenkel**

### CODE REVIEW

**Comment by zeke** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -173,7 +170,7 @@ Let's take our application for a spin again by running `npm start` from the comm
 
 Let's actually load some content, shall we?
 
-Make sure you require the `path` module towards the beginning of `main.js`:
+Make sure you require the [`path`](http://electron.atom.io/docs/api/path/) module towards the beginning of `main.js`:
```
Comment (feross) [inline README.md:173]: This is actually a node.js module: https://nodejs.org/api/path.html

File: README.md [hunk 2]
```diff
@@ -295,7 +292,7 @@ function openFile () {
 
 Now that we have the location of our file, let's read from that location. `fs.readFileSync` returns a `Buffer` object. We know we're working with text. So, we'll turn that into a string using the `toString()` method.
 
-Make sure you require the `fs` module towards the beginning of `main.js`:
+Make sure you require the [`fs`](http://electron.atom.io/docs/api/fs/) module towards the beginning of `main.js`:
```
Comment (feross) [inline README.md:295]: Same: https://nodejs.org/api/fs.html

**Review by feross — APPROVED** [review]
LGTM, I'll fix the two links I mentioned

**Pull request merged**